src/routes/(console)/onboarding/create-project/+page.svelte

```diff
@@ -79,7 +79,7 @@
             alt="Appwrite Logo" />
         <Card.Base variant="primary" padding="l">
             <CreateProject
-                regions={$regionsStore.regions}
+                regions={$regionsStore?.regions ?? []}
```

Comment:
can we just fix initialize the store so we never deal with any null checks? https://github.com/appwrite/console/pull/1919